x/qgb/abci.go

```diff
@@ -114,3 +122,67 @@ func handleValsetRequest(ctx sdk.Context, k keeper.Keeper) {
 		}
 	}
 }
+
+// maybePruneAttestations runs basic checks on saved attestations to see if we need to prune or not.
+// Then, it starts pruning the expired attestations until all attestations in state are still valid.
+func maybePruneAttestations(ctx sdk.Context, k keeper.Keeper) {
```

Comment:
[not blocking]
I think its a bit clearer to just have this named to pruneAttestations since there is only one situation in which we don't prune. Then simply say in the docs that this func prunes all attestations that have surpassed the expiry point, which is a very accurate description of what is occuring

Reply:
[23aac61](https://github.com/celestiaorg/celestia-app/pull/1799/commits/23aac61c090fdab90631f7d7d24488bcb8a0c27b)